boba-chain-ops/genesis/dump.go

```diff
@@ -0,0 +1,60 @@
+package genesis
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/trie"
+)
+
+type GenesisConfig struct {
+	db            ethdb.Database
+	outputPath    string
+	hardforkBlock int64
+}
+
+func NewGenesisConfig(db ethdb.Database, outputPath string, hardforkBlock int64) *GenesisConfig {
+	return &GenesisConfig{
+		db:            db,
+		outputPath:    outputPath,
+		hardforkBlock: hardforkBlock,
+	}
+}
+
+func (cfg *GenesisConfig) Dump() error {
+	hash := rawdb.ReadHeadHeaderHash(cfg.db)
+	if cfg.hardforkBlock == 0 {
```

Comment:
Shouldn't this be "!= 0"?

Reply:
Yes! This PR can close. I can write a simple go script to dump state to json. `geth dump` doesn't print out json.